MESC_Common/Src/MESCuart.c

```diff
@@ -34,18 +34,29 @@
 #include "MESCfoc.h"
 #include "MESCmotor_state.h"
 
-extern UART_HandleTypeDef huart3;
+#ifdef USB
```

Comment:
Maybe a more verbose define such as `MESC_UART_USB`?

Reply:
You are right. For the future we should make it possible to define multiple connections. That would need a change to the MESCcli to give the functions a cli handle or so